Automattic/wp-calypso
# Site Transfer: improve the action panel style

- Related to https://github.com/Automattic/wp-calypso/pull/77216

## Proposed Changes

* Remove the overflow hidden for all the `action-panel__body`
* Add space below the form for site transfer panel

## Testing Instructions



- See testing instructions on https://github.com/Automattic/wp-calypso/pull/77216
- Test other `.action-panel__body` to confirm they still looking good. E.g. `/settings/start-over/{$siteSlug}`

## Screenshots

| **Before** | **After** |
|---|---|
| <img width="727" alt="Screenshot 2023-05-24 at 12 12 22" src="https://github.com/Automattic/wp-calypso/assets/779993/d0f053d9-817a-465b-9e85-659118b2f95c"> | <img width="738" alt="Screenshot 2023-05-24 at 12 11 27" src="https://github.com/Automattic/wp-calypso/assets/779993/ec8b0c86-4fba-4581-8faa-8d97ee3cf962"> |

## Pre-merge Checklist
